atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -28,7 +28,11 @@ class CraneTimeoutError(Exception):
     pass
 
 
-class PulpPullPlugin(PostBuildPlugin):
+# Note: We use multiple inheritance here only to make it explicit that
```

Comment:
Ah right, misread the diff. Thought it was already inheriting from `ExitPlugin`.